tldw_Server_API/app/main.py

```
@@ -4646,6 +4659,26 @@ async def _claims_rebuild_loop():
         await shutdown_all_audit_services()
         logger.info("App Shutdown: Unified audit services stopped")
+
+        try:
+            from tldw_Server_API.app.core.Embeddings.audit_adapter import (
+                shutdown_local_audit_adapter_loop,
+            )
+
+            shutdown_local_audit_adapter_loop()
+            logger.info("App Shutdown: Embeddings audit adapter loop stopped")
+        except _STARTUP_GUARD_EXCEPTIONS as _e:
+            logger.debug(f"Embeddings audit adapter loop shutdown skipped: {_e}")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)